bokehjs/src/coffee/models/widgets/data_table.coffee

```diff
@@ -149,9 +149,15 @@ export class DataTableView extends WidgetView
     if @model.row_headers
       columns.unshift(@newIndexColumn())
 
+    reorderable = @model.reorderable
+
+    if reorderable and not $?.fn?.sortable?
+      logger.warn("jquery-ui is required to enable DataTable.reorderable")
+      reorderable = false
+
```

Comment:
This is about sorting rows, but reordering columns (aka. drag & drop). As said before, I will change slickgrid to not use ui for this, but as there is a quick workaround, this can be done in a future PR, so that we don't block this behemoth.